qa/tasks/mgr/dashboard/test_cephfs.py

```diff
@@ -9,6 +11,10 @@ class CephfsTest(DashboardTestCase):
 
     AUTH_ROLES = ['cephfs-manager']
 
+    def assertToHave(self, data, key):
```

Comment:
What about naming this `assertInAndNotNone`. It would be clear what it does without having to look up what it does.

Reply:
This code isn't part of this PR. This code has already be merged :/